src/Analyzers/Core/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -151,6 +151,8 @@ internal static class IDEDiagnosticIds
 
         public const string RemoveRedundantEqualityDiagnosticId = "IDE0100";
 
+        public const string RemoveUnnecessaryDiscardDesignationDiagnosticId = "IDE0110";
```

Comment:
Why are we spreading out the IDs with holes in the ranges?

Reply:
I notice the same for last 2 rules - I think we should rather keep these sequential as these are opaque IDs. I presume the gaps are introduced to account for similar rules that might come in future and this allows such rules to then slot in these gaps. I don't think we should do this as IDs are opaque numbers, and there are already large number of similar rules which have widely separate IDs.

Reply:
>  and there are already large number of similar rules which have widely separate IDs.

Yes.  And this has been unpleasant :)   For example, not being able to keep the rules organized here in linear order.  Needing them to be spread out randomly in user files, etc.